cmake/HPX_SetupTarget.cmake

```diff
@@ -187,9 +187,8 @@ function(hpx_setup_target target)
     set(_USE_CONFIG 0)
   endif()
 
-  # linker instructions
   if(NOT target_NOLIBS)
-    set(hpx_libs hpx)
+    set(hpx_libs hpx ${hpx_libs})
     if(NOT target_STATIC_LINKING)
       set(hpx_libs ${hpx_libs})
```

Comment:
Can we remove `${hpx_libs}` here now?

Reply:
oh, I forgot to remove it from there. I'll do it now